apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -233,7 +233,7 @@ defmodule Aecore.Miner.Worker do
       valid_txs_by_chainstate = BlockValidation.filter_invalid_transactions_chainstate(ordered_txs_list, chain_state, top_block.header.height + 1)
       valid_txs_by_fee = filter_transactions_by_fee(valid_txs_by_chainstate)
 
-      {_, pubkey} = Keys.pubkey()
+      pubkey = Wallet.get_public_key(Wallet.get_aewallet_dir())
```

Comment:
I think if you are calling internal function to get the path then better do it inside the Wallet.